pkg/virt-controller/services/renderresources_test.go

```diff
@@ -208,13 +207,13 @@ var _ = Describe("Resource pod spec renderer", func() {
 				func(fullPCPUEnabled, defineUserSpecifiedCPULimit bool, cores uint32, expectedCPUOverhead string) {
 					cpuIsolatedEmulatorThreadOverhead := resource.MustParse(expectedCPUOverhead)
 					var userSpecifiedCPULimit kubev1.ResourceList
-					var cpuManagerPolicyBetaOption v1.CPUManagerPolicyBetaOptions
+					vmiAnnotations := map[string]string{}
```

Comment:
Please consider passing the annotations map to the table function instead of a boolean.
It has a potential to remove an if statement and to improve readability.

Reply:
DONE